frontend/src/pages/EventSignInPage/index.tsx

```diff
@@ -58,15 +58,15 @@ function EventSignInPage(): JSX.Element {
 
           <Grid item>
             <EventSignInForm
-              handleSubmit={(values: AppUserEventRequest) =>
-                signInToEvent(eventID, values)
-              }
-            />
+              handleSubmit={(values: AppUserEventRequest, callback) =>
+                signInToEvent(eventID, values).then().catch((error) => {
+                  callback((error.status === 403) || (error.status === 500));
```

Comment:
What did you do to get status code 500 for the response? Looking at the endpoint handler and stuff in the backend, it should be returning 404 and stuff, so I'm just curious as to how you're getting 500s